website/docs/r/rde_type_behavior.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_rde_type_behavior"
+sidebar_current: "docs-vcd-resource-rde-type-behavior"
+description: |-
+   Provides the capability of managing RDE Type Behaviors in VMware Cloud Director.
+---
+
+# vcd\_rde\_type\_behavior
+
+Provides the capability of managing RDE Type Behaviors in VMware Cloud Director, which override an existing [RDE Interface
+Behavior](/providers/vmware/vcd/latest/docs/resources/rde_interface_behavior).
+
+Supported in provider *v3.10+*. Requires System administrator privileges.
+
+## Example Usage With Execution Override
+
+```hcl
+data "vcd_rde_interface" "my_interface" {
+  vendor  = "bigcorp"
+  nss     = "tech1"
+  version = "1.2.3"
+}
+
+resource "vcd_rde_interface_behavior" "my_interface_behavior" {
+  rde_interface_id = vcd_rde_interface.my_interface.id
+  name             = "MyBehavior"
+  description      = "Adds a node to the cluster.\nParameters:\n  clusterId: the ID of the cluster\n  node: The node address\n"
+  execution = {
+    "id" : "addNode"
+    "type" : "Activity"
+  }
+}
+
+resource "vcd_rde_type" "my_rde_type" {
+  vendor        = "vmware"
+  nss           = "vcd"
+  version       = "4.5.6"
+  name          = "My VMware RDE Type"
+  interface_ids = [data.vcd_rde_interface.my_interface.id]
+  schema        = file("${path.module}/schemas/my-type-schema.json")
+
+  # Behaviors can't be created after the RDE Interface is used by a RDE Type
+  # so we need to depend on the Behavior to wait for it to be created first.
+  depends_on = [vcd_rde_interface_behavior.my_interface_behavior]
+}
+
+resource "vcd_rde_type_behavior" "my_rde_type_behavior" {
+  rde_type_id               = vcd_rde_type.my_rde_type.id
+  rde_interface_behavior_id = vcd_rde_interface_behavior.my_interface_behavior.id
+  execution = {
+    "id" : "addNodeOverrided"
+    "type" : "Activity"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `rde_type_id` - (Required) The ID of the RDE Type that owns the Behavior
+* `rde_interface_behavior_id` - (Required) The ID of the [RDE Interface Behavior](/providers/vmware/vcd/latest/docs/resources/rde_interface_behavior) to override
+* `description` - (Optional) The description of the RDE Type Behavior.
+* `execution` - (Required) A map that specifies the Behavior execution mechanism.
+  You can find more information about the different execution types, like `WebHook`, `noop`, `Activity`, `MQTT`, `VRO`, `AWSLambdaFaaS`
+  and others [in the Extensibility SDK documentation](https://vmware.github.io/vcd-ext-sdk/docs/defined_entities_api/behaviors)
+
+## Attribute Reference
+
+* `name` - Name of the overridden Behavior
+* `ref` - The Behavior invocation reference to be used for polymorphic behavior invocations
```

Comment:
Similar query to there https://github.com/vmware/terraform-provider-vcd/pull/1074/files#diff-8e094640ce4dd5e7da41399262702d7129f7443f9db76f8db95fcbd7d928cdf0R72

Would this not be a polymorphic behavior anymore as it is on exact `vcd_rde_type`?

Reply:
In this case, `ref` would be the Interface Behavior ID. It is like the "original" reference. I can rephrase the sentence, as I just used the official one.